src/Test/Perf/util/runner_util.csx

```diff
@@ -38,3 +40,134 @@ async Task<ScriptState<object>> RunFile(string fileName)
     return await state.ContinueWithAsync<object>(text, scriptOptions);
 }
 
+/// Gets all csx file recursively in a given directory
+IEnumerable<string> GetAllCsxRecursive(string directoryName)
```

Comment:
Then we should just delete ```AllCsiRecursive``` and the skip folder stuff (I don't think it's called anywhere else).